deploy.tf

```diff
@@ -167,7 +167,7 @@ resource "digitalocean_droplet" "k8s_worker" {
         inline = [
             "chmod +x /tmp/install-kubeadm.sh",
             "sudo /tmp/install-kubeadm.sh",
-            "export NODE_PRIVATE_IP=\"${self.ipv4_address}\"",
+            "export NODE_PRIVATE_IP=\"${self.ipv4_address_private}\"",
```

Comment:
@bhcleek , did you test this? I believe this will cause inter-cluster communication to fail due to flannel being configured to use the public network.

Reply:
I did teato, but it's possible I missed something. Feel free to submit a patch if you find a reproducible problem.